t/venom/pfservers/backup_db_and_restore/00_backup_db_and_restore.yml

```diff
@@ -0,0 +1,78 @@
+name: Backup db and restore 
+version: 2
+testcases:
+- name: create_users
+  steps:
+    - type: http
+      method: POST
+      url: '{{.pfserver_webadmin_url}}/api/v1/users'
+      ignore_verify_ssl: true
+      body: >-
+        {
+          "pid_overwrite": "0",
+          "pid": "{{.backup_db_and_restore.create_user}}",
```

Comment:
Only `pid` is necessary to create a user, you can remove all other things.

Reply:
done